app/serializers/rest/instance_serializer.rb

```diff
@@ -24,6 +24,12 @@ def version
     Mastodon::Version.to_s
   end
 
+  def stats
+    { user_count:   User.confirmed.count,
```

Comment:
Aha. I'm not sure I understand what I have just done, but I pushed a new version of the PR that uses `InstancePresenter`, and appears to work too.

:crossed_fingers: